test/e2e/lib/pages/reader-page.js

```diff
@@ -26,6 +26,14 @@ export default class ReaderPage extends AsyncBaseContainer {
 		return URL.parse( href ).host;
 	}
 
+	async shareLatestPost() {
+		await driverHelper.clickWhenClickable( this.driver, by.css( '.reader-share__button' ) );
+		return await driverHelper.clickWhenClickable(
+			this.driver,
+			by.css( '.reader-popover .site__content' )
+		);
+	}
+
```

Comment:
In case if it happens, that e2e user doesn't follow any site and the reader is empty, this will fail. What do you think to put this in `try` block and throw an error?